src/Database.hs

```diff
@@ -0,0 +1,61 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Database where
+
+import           Data.Text                  (Text)
+import           Database.PostgreSQL.Simple (ConnectInfo (..), Connection,
+                                             close, connect, execute, query_)
+
+import           System.Environment         (getEnv)
+
+type URL = String
```

Comment:
Inconsistent with our table, url is TEXT, like title and description ?

Reply:
in Postgresql yes @hussein-aitlahcen but not in our type description of `Video` in Haskell :)

Reply:
@paulRbr I meant that he used Text for title and description aswell in the code. So why switch to string for the url only ? Maybe I'm missing something xD

Reply:
In PG you want to store everything that's a string-like to TEXT unless you have a very good reason to do otherwise.

In Haskell you have the alternative of String or Text, and in this case I choose to use String for URLs since youtube urls are straightword strings without any fancy characters. Title and descriptions need encoding support.

Anyway it can be changed freely.